Single deploy script

jhedstrom opened the pull request:
The goal here is to have a single deploy script for dev/test/production.

LP: https://app.liquidplanner.com/space/52157/projects/show/13954063P

jhedstrom committed:
Single deploy script.

- Removes production deployment script in preparation for a single
  script.

jhedstrom committed:
Some initial help text and configuration work for deploy script.

mariacha commented on bin/deploy:
```diff
@@ -2,34 +2,87 @@
+#
+show_help() {
+cat <<EOF
+  Usage: bin/rebuild [options] {dev|test|production}
```
Should this be bin/deploy?

jhedstrom replied:
Oh, I suppose if we want accurate help text =)

jhedstrom committed:
Use proper script name.

jhedstrom committed:
Deploy script now solely deals with deployment.

jhedstrom commented:
@thrnio I started to remove any notion of environment config, but  then realized the available environments would not be self documenting if we went that route.

Commit:
Update bin/deploy to check for $PORT

Commit:
bin/deploy to not use an environment conf-file

Now the script uses environment variables that are just available to the
script, instead of loading a specified environment file. This makes it
easier to configure for arbitrary environments.
